tekton/ci/plumbing-template.yaml

```diff
@@ -85,8 +400,6 @@ spec:
       pipelineRef:
         name: tekton-random-check
       params:
-        - name: message
-          value: "BuildUUID#: $(params.buildUUID), Package: $(params.package), Repo: $(params.gitRepository), Revision: $(params.gitRevision)"
```

Comment:
Did you mean to remove this from the `tekton-random-check`?

Reply:
I can :wink: